packages/main/src/themes/SplitButton.css

```diff
@@ -308,111 +254,66 @@
 	position: absolute;
 	box-sizing: border-box;
 	pointer-events: none;
-	left: var(--_ui5_split_button_middle_separator_left);
-	top: var(--_ui5_split_button_middle_separator_top);
-	right: 0;
+	width: 0.0625rem;
+	background-color: var(--sapButton_TextColor);
+	inset-inline-start: var(--_ui5_split_button_middle_separator_left);
+	inset-block-start: var(--_ui5_split_button_middle_separator_top);
 	height: var(--_ui5_split_button_middle_separator_height);
-	border: 0 solid var(--sapButton_TextColor);
-	border-left-width: 0.0625rem;
 }
 
 .ui5-split-arrow-button[design="Emphasized"]:before {
 	content: "";
 	position: absolute;
 	box-sizing: border-box;
 	pointer-events: none;
-	left: var(--_ui5_split_button_middle_separator_left);
-	top: var(--_ui5_split_button_middle_separator_top);
-	right: 0;
+	inset-inline-start: var(--_ui5_split_button_middle_separator_left);
+	inset-block-start: var(--_ui5_split_button_middle_separator_top);
+	inset-inline-end: 0;
 	height: var(--_ui5_split_button_middle_separator_height);
-	border: 0 solid var(--sapButton_TextColor);
-	border-left-width: 0.0625rem;
+	border: 0 solid var(--sapButton_Emphasized_TextColor);
+	border-inline-start-width: 0.0625rem;
```

Comment:
I think that here should be width instead of border, since we are not using the border property of before element. Also please fix the pseudo element by adding second `:` (applies for other repetitions in the file)